### DISCUSSION TITLE
Do I need to be using start& over start? I couldn't figure out what "&" is doing.

### DISCUSSION THREAD

**Discussion by sdpools** [tagyoureit/nodejs-poolController#409] [App and O/S Config Questions] [answered]
Hello,

Do I need to be using npm start& over npm start? I couldn't figure out what "&" is doing.

Thank you

**Comment by tagyoureit** [comment, marked as answer]
I answered this in https://github.com/tagyoureit/nodejs-poolController/discussions/408#discussioncomment-2541973.
